sdk/java-sdk/src/main/java/kalix/javasdk/workflowentity/WorkflowEntity.java

```diff
@@ -171,6 +189,9 @@ protected final Effect.Builder<S> effects() {
     interface ErrorEffect<T> extends Effect<T> {
     }
 
+    /**
+     * A workflow effect type that contains information about transition to next step. This could be also special transition to pause or end the workflow.
```

Comment:
```suggestion
     * A workflow effect type that contains information about the transition to the next step. This could be also a special transition to pause or end the workflow.
```